pkg/scyllaclient/client.go

```diff
@@ -158,7 +158,9 @@ func (c *Client) Status(ctx context.Context, host string) (NodeStatusInfoSlice,
 }
 
 func (c *Client) Decommission(ctx context.Context, host string) error {
-	_, err := c.scyllaOps.StorageServiceDecommissionPost(&scyllaOperations.StorageServiceDecommissionPostParams{Context: forceHost(ctx, host)})
+	queryCtx := forceHost(ctx, host)
+	queryCtx = noRetry(queryCtx)
```

Comment:
Please add a comment explaining why retry is disabled - unfortunate 500 response.

Reply:
added